apps/aecore/lib/aecore/structures/voting_question_tx.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Aecore.Structures.VotingQuestionTx do
+  alias Aecore.Structures.VotingQuestionTx
+  alias Aecore.Chain.Worker, as: Chain
+
+  require Logger
+
+  @type t :: %VotingQuestionTx{
+          question: String.t(),
+          possible_answer_count: non_neg_integer(),
```

Comment:
as I understand that limits the total count of answers, I think it is not needed, as it would only result in unfair votes